src/input/Input.tsx

```diff
@@ -14,12 +14,19 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-import React, { forwardRef } from "react";
+import React, { forwardRef, ReactNode } from "react";
 import classNames from "classnames";
+
 import styles from "./Input.module.css";
 import { ReactComponent as CheckIcon } from "../icons/Check.svg";
 
-export function FieldRow({ children, rightAlign, className, ...rest }) {
+interface FieldRowProps {
+  children: JSX.Element;
+  rightAlign?: boolean;
+  className?: string;
+}
+
+export function FieldRow({ children, rightAlign, className }: FieldRowProps) {
```

Comment:
Please add a return type

Reply:
I did not add the return types to components for most of the classes since they are determined automatically.
Should we change that in all files?